esp-hal/src/rtc_cntl/mod.rs

```diff
@@ -201,14 +208,35 @@ impl<'d> Rtc<'d> {
         #[cfg(any(esp32, esp32s3, esp32c3, esp32c6))]
         RtcSleepConfig::base_settings(&this);
 
+        if let Some(interrupt) = interrupt {
+            unsafe {
+                crate::interrupt::bind_interrupt(
```

Comment:
Can you please import the `crate::interrupt` module and the `crate::peripherals::Interrupt` type? There's no reason to include all these paths here, just makes it harder to read.

Reply:
Sure!